osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs

```diff
@@ -80,7 +75,7 @@ protected override void Update()
 
             // position updates should not occur if the item is filtered away.
             // this avoids panels flying across the screen only to be eventually off-screen or faded out.
-            if (!Item.Visible)
+            if (!Item!.Visible)
```

Comment:
I'll just highlight this one as a particularly egregious example (`!Item!`??? that is illegible)

Reply:
These should use `Debug.Assert` if anything.